receipt/SaleReceipt_de_CH.tpl

```diff
@@ -0,0 +1,1365 @@
+				{{ _self.layaways(Sale.Customer,Sale.isTaxInclusive,parameters.gift_receipt,options)}}
+			{% endif %}
+			{% if options.show_customer_specialorders %}
+				{{ _self.specialorders(Sale.Customer,Sale.isTaxInclusive,parameters.gift_receipt,options)}}
+			{% endif %}
+			{% if options.show_customer_workorders %}
+				{{ _self.workorders(Sale.Customer,parameters.gift_receipt,options)}}
+			{% endif %}
+		{% endif %}
+
+		{% if options.show_customer_credit_account and Sale.Customer and not parameters.gift_receipt and not store_copy %}
+			{% if Sale.Customer.CreditAccount and Sale.Customer.CreditAccount.MetaData.creditBalanceOwed > 0 or Sale.Customer.CreditAccount.MetaData.extraDeposit > 0 %}
+				<h2 class="footerSectionTitle">Kundenkonto</h2>
+				<table class="totals">
+					{% if Sale.Customer.CreditAccount.MetaData.creditBalanceOwed > 0 %}
+						<tr>
+							<td width="100%">Geschuldeter Saldo: </td>
+							<td class="amount">{{ Sale.Customer.CreditAccount.MetaData.creditBalanceOwed|money }}</td>
+						</tr>
+					{% elseif Sale.Customer.CreditAccount.MetaData.extraDeposit > 0 %}
+						<tr>
+							<td width="100%">Gegen Einzahlung: </td>
+							<td class="amount">{{ Sale.Customer.CreditAccount.MetaData.extraDeposit|money }}</td>
+						</tr>
+					{% endif %}
+				</table>
+			{% endif %}
+			{% if Sale.Customer.MetaData.getAmountToCompleteAll > 0 %}
+				<table class="totals">
+					<tr class="total">
+						<td width="100%">Verbleibender Saldo: </td>
+						<td class="amount">{{ Sale.Customer.MetaData.getAmountToCompleteAll|money }}</td>
+					</tr>
+				</table>
+			{% endif %}
+		{% endif %}
+	{% endif %}
+{% endmacro %}
+
+{% macro cc_payment_info(Sale,Payment) %}
+	<tr>
+		{% if Payment.archived == 'false' %}
+			{% if Payment.ccChargeID > 0 and Payment.CCCharge.declined == 'false' %}
+				<td class="label" width="100%">
+					{{ Payment.PaymentType.name }}
+					<br>Kartennummer: {{Payment.CCCharge.xnum}}
+					{% if Payment.CCCharge.cardType|strlen > 0 %}
+						<br>Typ: {{Payment.CCCharge.cardType}}
+					{% endif %}
+					{% if Payment.CCCharge.cardholderName|strlen > 0 %}
+						<br>Karteninhaber: {{Payment.CCCharge.cardholderName}}
+					{% endif %}
+					{% if Payment.CCCharge.entryMethod|strlen > 0 %}
+						<br>Eingabe: {{Payment.CCCharge.entryMethod}}
+					{% endif %}
+					{% if Payment.CCCharge.authCode|strlen > 0 %}
+						<br>Freigabe: {{Payment.CCCharge.authCode}}
+					{% endif %}
+					{% if Payment.CCCharge.gatewayTransID|strlen > 0 and Payment.CCCharge.gatewayTransID|strlen < 48 %}
+						<br>ID: {{Payment.CCCharge.gatewayTransID}}
+					{% endif %}
+					{% if Payment.CCCharge.MetaData.isEMV %}
+						{% if Payment.CCCharge.MetaData.AID|strlen > 0 %}
+							<br>ID der Applikation: {{Payment.CCCharge.MetaData.AID}}
+						{% endif %}
+						{% if Payment.CCCharge.MetaData.ApplicationLabel|strlen > 0 %}
+							<br>Applikationsaufkleber: {{Payment.CCCharge.MetaData.ApplicationLabel}}
```

Comment:
Got the correct translation for Application Label: Please replace _Applikationsaufkleber_ with _Applikationsbezeichnung_

Reply:
Done